### DISCUSSION TITLE
Allow for setting the "Reporter" field on issue creation

### DISCUSSION THREAD

**Discussion by openfellas-freiburg** [ankitpokhrel/jira-cli#189] [Ideas] [answered]
Currently it is only possible to set the assignee, while the reporter will always be set to the current user.
It would be very helpful for scripts to be able to also set the "Reporter" field on issue creation.

Thank you very much for your work.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @axxelia-freiburg, this is done with #539 and will be available in the next release. If you wish to use it for now, you can do so by installing the main branch using one of the following methods.

```sh
# Docker
$ docker run -it --rm ghcr.io/ankitpokhrel/jira-cli:main

# Golang
$ go install github.com/ankitpokhrel/jira-cli/cmd/jira@main

# Homebrew
$ brew tap ankitpokhrel/jira-cli
$ brew install jira-cli --HEAD
```

You can also compile the code locally using the steps [mentioned here](https://github.com/ankitpokhrel/jira-cli#development).

**Reply by ankitpokhrel** [reply]
This feature is available since the latest release [v1.3.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.3.0)
